.codeclimate.yml

```diff
@@ -0,0 +1,47 @@
+version: "2"         # required to adjust maintainability checks
+checks:
+  argument-count:
+    config:
+      threshold: 4
+  complex-logic:
+    config:
+      threshold: 4
+  file-lines:
+    config:
+      threshold: 250
+  method-complexity:
+    config:
+      threshold: 5
+  method-count:
+    config:
+      threshold: 20
+  method-lines:
+    config:
+      threshold: 25
+  nested-control-flow:
+    config:
+      threshold: 4
+  return-statements:
+    config:
+      threshold: 4
+  similar-code:
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+  identical-code:
+    config:
+      threshold: # language-specific defaults. an override will affect all languages.
+plugins:
+  rubocop:
+    enabled: true
+    channel: rubocop-0-52
+  brakeman:
+    enabled: true
+  bundler-audit:
+    enabled: true
+exclude_patterns:
+  - 'config/'
+  - 'db/'
+  - 'features/'
```

Comment:
This isn't part of our structure

Reply:
it was part of the defaults, but I'll take it out.  At first I was thinking this WIP was using it https://github.com/department-of-veterans-affairs/vets-api/pull/3102